src/core/utils/toast/toast.less

```diff
@@ -0,0 +1,54 @@
+@import '~common/less/mixins.less';
+@import '~common/less/constants.less';
+@import '~common/less/colors.less';
+
+.toastContainer {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  box-sizing: border-box;
+  border: 8px solid transparent;
```

Comment:
The toast-messages should have a border-radius of 3px (same as .owCard())

Reply:
This border is not visible. It is used as used as the margin around the entire list of messages.

The reason for not using a margin is that the content would overflow inside the container as it is set to a width of `100vw`, which a margin would not uphold.